### PR TITLE
Confirmed api toggle

### PR DESCRIPTION
**Pull request by lewey**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala [hunk 1]
```diff
@@ -73,16 +73,21 @@ class BbsiUrls @Inject()(config: DesConfig) {
...
 
 @Singleton
-class TaxAccountUrls @Inject()(npsConfig: NpsConfig, desConfig: DesConfig) {
+class TaxAccountUrls @Inject()(npsConfig: NpsConfig, desConfig: DesConfig, featureTogglesConfig: FeatureTogglesConfig) {
 
-  def taxAccountUrlNps(nino: Nino, taxYear: TaxYear): String =
-    s"${npsConfig.baseURL}/person/${nino.nino}/tax-account/${taxYear.year}/calculation"
+  private val desTaxAccountURL = (nino: Nino, taxYear: TaxYear) =>
+    s"${desConfig.baseURL}/pay-as-you-earn/individuals/${nino.nino}/tax-account/tax-year" +
+      s"/${taxYear.year}${if(!featureTogglesConfig.confirmedAPIEnabled)"?calculation=true" else ""}"
```
Comment (oc-ste) [inline app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala]: Don't need the empty else here

File: app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala [hunk 2]
```diff
@@ -73,16 +73,21 @@ class BbsiUrls @Inject()(config: DesConfig) {
...
 
-  def taxAccountUrlDes(nino: Nino, taxYear: TaxYear): String =
-    s"${desConfig.baseURL}/pay-as-you-earn/individuals/${nino.nino}/tax-account/tax-year/${taxYear.year}?calculation=true"
+  private val npsTaxAccountURL = (nino: Nino, taxYear: TaxYear) =>
+    s"${npsConfig.baseURL}/person/${nino.nino}/tax-account/${taxYear.year}${if(!featureTogglesConfig.confirmedAPIEnabled)"/calculation" else ""}"
```
Comment (oc-ste) [inline app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala]: Same as above

File: app/uk/gov/hmrc/tai/connectors/TaxAccountConnector.scala [hunk 3]
```diff
@@ -42,24 +42,20 @@ class TaxAccountConnector @Inject()(npsConfig: NpsConfig,
...
-    if(featureTogglesConfig.desEnabled) {
-      implicit val hc: HeaderCarrier = createHeader.withExtraHeaders("Gov-Uk-Originator-Id" -> desConfig.originatorId)
-      val url = taxAccountUrls.taxAccountUrlDes(nino, taxYear)
-      httpHandler.getFromApi(url, APITypes.DesTaxAccountAPI)
-    }
-    else{
-      val hcWithHodHeaders = hc.withExtraHeaders("Gov-Uk-Originator-Id" -> npsConfig.originatorId)
-      val url = taxAccountUrls.taxAccountUrlNps(nino, taxYear)
-      httpHandler.getFromApi(url, APITypes.NpsTaxAccountAPI)(hcWithHodHeaders)
+  def hcWithHodHeaders(implicit hc:HeaderCarrier) =
```
Comment (oc-ste) [inline app/uk/gov/hmrc/tai/connectors/TaxAccountConnector.scala]: Missing return type

File: app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala (hunk 1 above)
Comment (lewey) [reply app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala]: I think the if is evaluation to an expression

File: app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala (hunk 2 above)
Comment (lewey) [reply app/uk/gov/hmrc/tai/connectors/ApplicationUrls.scala]: I think the if is evaluation to an expression

**Pull request merged**